gatsby-config.js

```diff
@@ -19,6 +19,13 @@ module.exports = {
     },
     plugins: [
         'gatsby-plugin-react-helmet',
+        {
+            resolve: `gatsby-plugin-react-helmet-canonical-urls`,
```

Comment:
Hello, @yauheni-kryzhyk-deriv just curious, what does this config do?

Reply:
Hello, @alamin-deriv, it creates <link rel="canonical" href="...." /> for seo optimization.
you can check here: https://www.gatsbyjs.com/plugins/gatsby-plugin-react-helmet-canonical-urls/

Reply:
Cool, thanks man.